pkg/compiler/codegen.go

```diff
@@ -1828,27 +1825,17 @@ func (c *codegen) convertStruct(lit *ast.CompositeLit, ptr bool) {
 	}
 
 	emit.Opcodes(c.prog.BinWriter, opcode.NOP)
```

Comment:
@fyrchik, btw, what do you think about this NOP?

Reply:
I've traced it back to 8fe079ec8ebb83ce53d6f1535bfe513271231958.

Reply:
I think, we can drop it. We insert it in some cases for easier `seq_points` and debugging, but this is not the case here. It may also be some N2 legacy.